.travis.yml

```diff
@@ -4,10 +4,14 @@ os:
   - linux
   - osx
```

Comment:
Would windows just be untested now?

Reply:
Yes, however we don't test Windows in any of our other packages.

I can include it here if you'd like, I just wanted to be more consistent with other packages which we maintain.

Reply:
Makes sense to test it for open source stuff in public orgs, IMO